lib/index.js

```diff
@@ -2,6 +2,9 @@
 
 var babel = require('@babel/core');
 
+var LocalCache = require('node-localcache');
```

Comment:
I'm reluctant to add [this package](https://github.com/glebmachine/node-localcache/) as a dependency. It has more dependencies with fixed versions and hasn't been maintained for 7 years.

Can caching be implemented without additional dependencies or a more popular and maintained package?

Reply:
Seconded.

The implementation of `node-localcache` writes all cache entries as one big JSON object to a single file. This sounds bad for a preprocessor: if you're running Karma in watch mode, a small change to a single file inside a large project means (repeatedly) overwriting a massive JSON file. 😕

I would prefer an alternative caching solution that is more *gentle* with the file system.

Reply:
I inherited that from the original PR (#34).  Writing files without deps is not hard.  And writing to individual files instead of one big one is also easy.  I'll revise it.

Do you require old-style callbacks in this project or can I use fs/promises?

Reply:
Babel 7 supports Node 6 and up, which unfortunately means that both `fs.promises` and `util.promisify()` are off the table... 😞 (Or we'd have to wait for Babel 8...)